demo/webpack/webpack.config.js

```diff
@@ -62,7 +65,13 @@ module.exports = {
                 keepQuery: true,
               },
             },
-            'sass-loader?sourceMap',
+            {
+              loader: 'sass-loader',
+              options: {
+                sourceMap: true,
```

Comment:
Just wondering whether the demo is to be treated as a production-friendly example where the regular env variable `NODE_ENV === 'production` is to be respected to toggle the source maps?

Reply:
In a real project, you usually have `webpack.dev.js` and `webpack.prod.js`, where this kind of choice (to include the sourcemap or not) is made. I didn't want to overcomplicate the simple example here, that's all 👼